### ISSUE TITLE
After moving .fs file in fsproj code completion doesn't work

### ISSUE THREAD

**Issue by landy** [JetBrains/fsharp-support#147]
hello I've encountered pretty annoying issue:

When I add a new fs file to the project and then move it up in the list of files in fsproj I can't use the newly added code in the files that were previously on top of the new file but compilation works. After the restart of Rider everything works fine

this happens both in 2020.1.4 and 2020.2 EAP7 versions of Rider

**Comment by saul** [comment]
Already reported on YouTrack here: https://youtrack.jetbrains.com/issue/RIDER-46461

**Comment by auduchinok** [comment]
@landy Thanks for the report. I haven't seen this happening in 2020.1 builds, so there's probably something we're not aware of. What OS and MSBuild you use? Could you post your logs folder after reproducing it (here or to the YouTrack issue above)?

@saul It seems it could be something different, since what you're describing seems to be 2020.2-specific. I'm looking into it now.

**Comment by landy** [comment]
Thanks for investigation :) I've created a simple repo https://github.com/landy/rider-fileorder. The error doesn't occur only when adding new file. For me it happens even if I open this project and when I reorder the files like this 
```
<ItemGroup>
    <Compile Include="Database.fs" />
    <Compile Include="Domain.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>
```

Rider still thinks that the call in Domain.fs is invalid even if it can be compiled

**Comment by auduchinok** [comment]
Should be fixed in 2020.3.
